### PR TITLE
Implement uni-title (#43)

### PR DESCRIPTION
**Pull request by warpech**
Implements #43 

This is a general implementation of what was previously part of BlendingEditor (https://github.com/Starcounter/Blending/pull/104)

### CODE REVIEW

File: uniform.css [hunk 1]
```diff
@@ -468,3 +468,58 @@ Pagination
...
+
+/* title in a section */
+
+.uni-section-primary>.uni-title> ::slotted(*),
+.uni-section-secondary>.uni-title> ::slotted(*),
+.uni-section-tertiary>.uni-title> ::slotted(*),
+.uni-section-primary.uni-title> ::slotted(*),
+.uni-section-secondary.uni-title> ::slotted(*),
+.uni-section-tertiary.uni-title> ::slotted(*) {
+  margin: 7px 0 1px 0 !important;
```
Comment (alshakero) [inline uniform.css]: Is there a reason to have asymmetric margins for bottom and top? IMO it doesn't look very nice

image

It's not centered. 

[Here](http://qr.ae/TU8mf6) is the default margins for h1...h6. They all have symmetric margins.

File: uniform.css [hunk 2]
```diff
@@ -468,3 +468,58 @@ Pagination
...
+}
+
+.uni-section-primary>.uni-title>*,
+.uni-section-secondary>.uni-title>*,
+.uni-section-tertiary>.uni-title>*,
+.uni-section-primary.uni-title>*,
+.uni-section-secondary.uni-title>*,
+.uni-section-tertiary.uni-title>* {
+  margin: 7px 0 1px 0 !important;
```
Comment (alshakero) [inline uniform.css]: Same as above

File: uniform.css (hunk 1 above)
Comment (warpech) [reply uniform.css]: This is according to @andwah's design, so it is as desired.

**Comment by warpech** [comment]
Thanks, merging

**Pull request merged**